pkg/api/vlabs/types.go

```diff
@@ -125,6 +125,7 @@ type OrchestratorProfile struct {
 // Kubernetes specific configuration
 type KubernetesConfig struct {
 	KubernetesImageBase string `json:"kubernetesImageBase,omitempty"`
+	NetworkPolicy       string `json:"networkPolicy,omitempty"`
```

Comment:
Calico is not really used for any networking. When `networkPolicy=calico` it is still using the existing `kubenet` Pod network. The only thing that calico does is managing iptables rules to prevent unspecified network communication.

`NetworkConfig` IMHO should not be limited to `KubernetesConfig`, as other orchestrators potentially have similar integrations with the same overlay networks. I would suggest to change this as soon as is becomes necessary.